legacy-pages/import-wallet.html

```diff
@@ -22,6 +22,14 @@
   <script nonce="**CSP_NONCE**">
     window.NONCE = '**CSP_NONCE**'
   </script>
+  <script nonce="**CSP_NONCE**">
+    window.CAPTCHA_KEY = '**RECAPTCHA_KEY**'
+  </script>
+  <script
```

Comment:
Thanks [for the reference](https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/index.html#L25-L29) @schnogz

Reply:
you just combine the two lines into one

```
<script nonce="**CSP_NONCE**">
    window.NONCE = '**CSP_NONCE**'
  </script>
  <script nonce="**CSP_NONCE**">
    window.CAPTCHA_KEY = '**RECAPTCHA_KEY**'
  </script>
  ```
  to
  ```
<script nonce="**CSP_NONCE**">
    window.NONCE = '**CSP_NONCE**'
    window.CAPTCHA_KEY = '**RECAPTCHA_KEY**'
  </script>
  ```

Reply:
Makes sense! done!